extensions/wikia/AdEngine/js/lookup/rubicon/rubiconTier.js

```diff
@@ -38,8 +39,20 @@ define('ext.wikia.adEngine.lookup.rubicon.rubiconTier', [
 		return sizeId + '_tier' + math.leftPad(tier, 4);
 	}
 
-	function parsePrice(tier) {
-		var matches = /^\d+_tier(\d+)/g.exec(tier);
+	function parseOpenMarketPrice(tier) {
+		if (tier.indexOf(privateTierPostfix) === -1) {
+			var matches = /^\d+_tier(\d+)/g.exec(tier);
+
+			if (matches && matches[1]) {
+				return parseInt(matches[1], 10);
```

Comment:
do you mean something actionable? In case above `matches[1]` is the corresponding group `(\d+)` of regex which will always be a number